hexrd/transforms/new_capi/xf_new_capi.py

```diff
@@ -309,10 +317,7 @@ def xy_to_gvec(
     array_like, optional
         if output_ref is True
     """
-    # TODO: in the C library beam vector and eta vector are expected. However
-    # we receive rmat_b. Please check this!
```

Comment:
This function agrees with the numpy version, which takes rmat_b directly and does not do this decomposition.